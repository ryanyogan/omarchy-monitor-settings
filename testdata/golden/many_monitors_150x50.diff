Visual regression detected!

=== EXPECTED ===
  [90m╭──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮[0m    
  [90m│[0m                                                                                                                                              [90m│[0m    
  [90m│[0m                                                               [1;34mDisplay Settings[0m                                                               [90m│[0m    
  [90m│[0m                                                                                                                                              [90m│[0m    
  [90m╰──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯[0m    
                                                                                                                                                      
  [34m╭────────────────────────────────────────────────────────╮[0m  [36m╭──────────────────────────────────────────────────────────────────────────────────╮[0m    
  [34m│[0m                                                        [34m│[0m  [36m│[0m                                                                                  [36m│[0m    
  [34m│[0m                                                        [34m│[0m  [36m│[0m                                                                                  [36m│[0m    
  [34m│[0m  [1;34mNavigation[0m                                            [34m│[0m  [36m│[0m  [1;36mDisplay Overview[0m                                                                [36m│[0m    
  [34m│[0m                                                        [34m│[0m  [36m│[0m                                                                                  [36m│[0m    
  [34m│[0m  [1;34m▶[0m [1;34mDashboard[0m                                           [34m│[0m  [36m│[0m  [34m○[0m [1;32mHDMI-1[0m[1;33m 👆 CURRENT[0m                                                             [36m│[0m    
  [34m│[0m                                                        [34m│[0m  [36m│[0m  [37m  Dell U2414H-1[0m                                                                 [36m│[0m    
  [34m│[0m    [37mMonitor Selection[0m                                   [34m│[0m  [36m│[0m  [90m  1920x1080 @ 60Hz[0m                                                              [36m│[0m    
  [34m│[0m                                                        [34m│[0m  [36m│[0m  [90m  Scale: 1.0x[0m                                                                   [36m│[0m    
  [34m│[0m    [37mSmart Scaling[0m                                       [34m│[0m  [36m│[0m  [3;33m  → Scaling changes will apply here[0m                                             [36m│[0m    
  [34m│[0m                                                        [34m│[0m  [36m│[0m                                                                                  [36m│[0m    
  [34m│[0m    [37mManual Scaling[0m                                      [34m│[0m  [36m│[0m  [90m◦[0m [1;34mHDMI-2[0m                                                                        [36m│[0m    
  [34m│[0m                                                        [34m│[0m  [36m│[0m  [37m  Dell U2414H-2[0m                                                                 [36m│[0m    
  [34m│[0m    [37mSettings[0m                                            [34m│[0m  [36m│[0m  [90m  1920x1080 @ 60Hz[0m                                                              [36m│[0m    
  [34m│[0m                                                        [34m│[0m  [36m│[0m  [90m  Scale: 1.0x[0m                                                                   [36m│[0m    
  [34m│[0m    [37mHelp[0m                                                [34m│[0m  [36m│[0m                                                                                  [36m│[0m    
  [34m│[0m                                                        [34m│[0m  [36m│[0m  [90m◦[0m [1;33mHDMI-3[0m                                                                        [36m│[0m    
  [34m│[0m    [37mExit[0m                                                [34m│[0m  [36m│[0m  [37m  Dell U2414H-3[0m                                                                 [36m│[0m    
  [34m│[0m                                                        [34m│[0m  [36m│[0m  [90m  1920x1080 @ 60Hz[0m                                                              [36m│[0m    
  [34m│[0m                                                        [34m│[0m  [36m│[0m  [90m  Scale: 1.0x[0m                                                                   [36m│[0m    
  [34m│[0m                                                        [34m│[0m  [36m│[0m                                                                                  [36m│[0m    
  [34m│[0m                                                        [34m│[0m  [36m│[0m  [90m◦[0m [1;35mHDMI-4[0m                                                                        [36m│[0m    
  [34m│[0m                                                        [34m│[0m  [36m│[0m  [37m  Dell U2414H-4[0m                                                                 [36m│[0m    
  [34m│[0m                                                        [34m│[0m  [36m│[0m  [90m  1920x1080 @ 60Hz[0m                                                              [36m│[0m    
  [34m│[0m                                                        [34m│[0m  [36m│[0m  [90m  Scale: 1.0x[0m                                                                   [36m│[0m    
  [34m│[0m                                                        [34m│[0m  [36m│[0m                                                                                  [36m│[0m    
  [34m│[0m                                                        [34m│[0m  [36m│[0m                                                                                  [36m│[0m    
  [34m│[0m                                                        [34m│[0m  [36m│[0m                                                                                  [36m│[0m    
  [34m│[0m                                                        [34m│[0m  [36m│[0m                                                                                  [36m│[0m    
  [34m│[0m                                                        [34m│[0m  [36m│[0m                                                                                  [36m│[0m    
  [34m│[0m                                                        [34m│[0m  [36m│[0m                                                                                  [36m│[0m    
  [34m│[0m                                                        [34m│[0m  [36m│[0m                                                                                  [36m│[0m    
  [34m│[0m                                                        [34m│[0m  [36m│[0m                                                                                  [36m│[0m    
  [34m│[0m                                                        [34m│[0m  [36m│[0m                                                                                  [36m│[0m    
  [34m│[0m                                                        [34m│[0m  [36m│[0m                                                                                  [36m│[0m    
  [34m│[0m                                                        [34m│[0m  [36m│[0m                                                                                  [36m│[0m    
  [34m│[0m                                                        [34m│[0m  [36m│[0m                                                                                  [36m│[0m    
  [34m│[0m                                                        [34m│[0m  [36m│[0m                                                                                  [36m│[0m    
  [34m╰────────────────────────────────────────────────────────╯[0m  [36m╰──────────────────────────────────────────────────────────────────────────────────╯[0m    
                                                                                                                                                      
  [90m╭──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮[0m    
  [90m│[0m                                                                                                                                              [90m│[0m    
  [90m│[0m                                          [1;32m↑↓[0m  [37mnavigate[0m    [1;34m⏎[0m  [37mselect[0m    [1;33mh[0m  [37mhelp[0m    [1;35mesc[0m  [37mback[0m    [1;31mq[0m  [37mquit[0m                                        [90m│[0m    
  [90m│[0m                                                                                                                                              [90m│[0m    
  [90m╰──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯[0m    

=== ACTUAL ===
  ╭──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮    
  │                                                                                                                                              │    
  │                                                               Display Settings                                                               │    
  │                                                                                                                                              │    
  ╰──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯    
                                                                                                                                                      
  ╭────────────────────────────────────────────────────────╮  ╭──────────────────────────────────────────────────────────────────────────────────╮    
  │                                                        │  │                                                                                  │    
  │                                                        │  │                                                                                  │    
  │  Navigation                                            │  │  Display Overview                                                                │    
  │                                                        │  │                                                                                  │    
  │  ▶ Dashboard                                           │  │  ○ HDMI-1 👆 CURRENT                                                             │    
  │                                                        │  │    Dell U2414H-1                                                                 │    
  │    Monitor Selection                                   │  │    1920x1080 @ 60Hz                                                              │    
  │                                                        │  │    Scale: 1.0x                                                                   │    
  │    Smart Scaling                                       │  │    → Scaling changes will apply here                                             │    
  │                                                        │  │                                                                                  │    
  │    Manual Scaling                                      │  │  ◦ HDMI-2                                                                        │    
  │                                                        │  │    Dell U2414H-2                                                                 │    
  │    Settings                                            │  │    1920x1080 @ 60Hz                                                              │    
  │                                                        │  │    Scale: 1.0x                                                                   │    
  │    Help                                                │  │                                                                                  │    
  │                                                        │  │  ◦ HDMI-3                                                                        │    
  │    Exit                                                │  │    Dell U2414H-3                                                                 │    
  │                                                        │  │    1920x1080 @ 60Hz                                                              │    
  │                                                        │  │    Scale: 1.0x                                                                   │    
  │                                                        │  │                                                                                  │    
  │                                                        │  │  ◦ HDMI-4                                                                        │    
  │                                                        │  │    Dell U2414H-4                                                                 │    
  │                                                        │  │    1920x1080 @ 60Hz                                                              │    
  │                                                        │  │    Scale: 1.0x                                                                   │    
  │                                                        │  │                                                                                  │    
  │                                                        │  │                                                                                  │    
  │                                                        │  │                                                                                  │    
  │                                                        │  │                                                                                  │    
  │                                                        │  │                                                                                  │    
  │                                                        │  │                                                                                  │    
  │                                                        │  │                                                                                  │    
  │                                                        │  │                                                                                  │    
  │                                                        │  │                                                                                  │    
  │                                                        │  │                                                                                  │    
  │                                                        │  │                                                                                  │    
  │                                                        │  │                                                                                  │    
  │                                                        │  │                                                                                  │    
  ╰────────────────────────────────────────────────────────╯  ╰──────────────────────────────────────────────────────────────────────────────────╯    
                                                                                                                                                      
  ╭──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮    
  │                                                                                                                                              │    
  │                                          ↑↓  navigate    ⏎  select    h  help    esc  back    q  quit                                        │    
  │                                                                                                                                              │    
  ╰──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯    

=== DIFF INFO ===
Expected length: 11691
Actual length: 9760
Expected hash: 59a1cc4630f9c61eb01bd9187c7f5da57268858454a7f5295f9cb341cf1bbbfb
Actual hash: 3c0b72c5e6af73082d5e711b7f26e9bf7d9c93c7bb9904d574e9eee5c6c70e04
