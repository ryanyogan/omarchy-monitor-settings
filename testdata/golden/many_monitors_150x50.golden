# Visual Golden File
# Name: many_monitors
# Dimensions: 150x50
# Hash: 59a1cc4630f9c61eb01bd9187c7f5da57268858454a7f5295f9cb341cf1bbbfb

  [90m╭──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮[0m    
  [90m│[0m                                                                                                                                              [90m│[0m    
  [90m│[0m                                                               [1;34mDisplay Settings[0m                                                               [90m│[0m    
  [90m│[0m                                                                                                                                              [90m│[0m    
  [90m╰──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯[0m    
                                                                                                                                                      
  [34m╭────────────────────────────────────────────────────────╮[0m  [36m╭──────────────────────────────────────────────────────────────────────────────────╮[0m    
  [34m│[0m                                                        [34m│[0m  [36m│[0m                                                                                  [36m│[0m    
  [34m│[0m                                                        [34m│[0m  [36m│[0m                                                                                  [36m│[0m    
  [34m│[0m  [1;34mNavigation[0m                                            [34m│[0m  [36m│[0m  [1;36mDisplay Overview[0m                                                                [36m│[0m    
  [34m│[0m                                                        [34m│[0m  [36m│[0m                                                                                  [36m│[0m    
  [34m│[0m  [1;34m▶[0m [1;34mDashboard[0m                                           [34m│[0m  [36m│[0m  [34m○[0m [1;32mHDMI-1[0m[1;33m 👆 CURRENT[0m                                                             [36m│[0m    
  [34m│[0m                                                        [34m│[0m  [36m│[0m  [37m  Dell U2414H-1[0m                                                                 [36m│[0m    
  [34m│[0m    [37mMonitor Selection[0m                                   [34m│[0m  [36m│[0m  [90m  1920x1080 @ 60Hz[0m                                                              [36m│[0m    
  [34m│[0m                                                        [34m│[0m  [36m│[0m  [90m  Scale: 1.0x[0m                                                                   [36m│[0m    
  [34m│[0m    [37mSmart Scaling[0m                                       [34m│[0m  [36m│[0m  [3;33m  → Scaling changes will apply here[0m                                             [36m│[0m    
  [34m│[0m                                                        [34m│[0m  [36m│[0m                                                                                  [36m│[0m    
  [34m│[0m    [37mManual Scaling[0m                                      [34m│[0m  [36m│[0m  [90m◦[0m [1;34mHDMI-2[0m                                                                        [36m│[0m    
  [34m│[0m                                                        [34m│[0m  [36m│[0m  [37m  Dell U2414H-2[0m                                                                 [36m│[0m    
  [34m│[0m    [37mSettings[0m                                            [34m│[0m  [36m│[0m  [90m  1920x1080 @ 60Hz[0m                                                              [36m│[0m    
  [34m│[0m                                                        [34m│[0m  [36m│[0m  [90m  Scale: 1.0x[0m                                                                   [36m│[0m    
  [34m│[0m    [37mHelp[0m                                                [34m│[0m  [36m│[0m                                                                                  [36m│[0m    
  [34m│[0m                                                        [34m│[0m  [36m│[0m  [90m◦[0m [1;33mHDMI-3[0m                                                                        [36m│[0m    
  [34m│[0m    [37mExit[0m                                                [34m│[0m  [36m│[0m  [37m  Dell U2414H-3[0m                                                                 [36m│[0m    
  [34m│[0m                                                        [34m│[0m  [36m│[0m  [90m  1920x1080 @ 60Hz[0m                                                              [36m│[0m    
  [34m│[0m                                                        [34m│[0m  [36m│[0m  [90m  Scale: 1.0x[0m                                                                   [36m│[0m    
  [34m│[0m                                                        [34m│[0m  [36m│[0m                                                                                  [36m│[0m    
  [34m│[0m                                                        [34m│[0m  [36m│[0m  [90m◦[0m [1;35mHDMI-4[0m                                                                        [36m│[0m    
  [34m│[0m                                                        [34m│[0m  [36m│[0m  [37m  Dell U2414H-4[0m                                                                 [36m│[0m    
  [34m│[0m                                                        [34m│[0m  [36m│[0m  [90m  1920x1080 @ 60Hz[0m                                                              [36m│[0m    
  [34m│[0m                                                        [34m│[0m  [36m│[0m  [90m  Scale: 1.0x[0m                                                                   [36m│[0m    
  [34m│[0m                                                        [34m│[0m  [36m│[0m                                                                                  [36m│[0m    
  [34m│[0m                                                        [34m│[0m  [36m│[0m                                                                                  [36m│[0m    
  [34m│[0m                                                        [34m│[0m  [36m│[0m                                                                                  [36m│[0m    
  [34m│[0m                                                        [34m│[0m  [36m│[0m                                                                                  [36m│[0m    
  [34m│[0m                                                        [34m│[0m  [36m│[0m                                                                                  [36m│[0m    
  [34m│[0m                                                        [34m│[0m  [36m│[0m                                                                                  [36m│[0m    
  [34m│[0m                                                        [34m│[0m  [36m│[0m                                                                                  [36m│[0m    
  [34m│[0m                                                        [34m│[0m  [36m│[0m                                                                                  [36m│[0m    
  [34m│[0m                                                        [34m│[0m  [36m│[0m                                                                                  [36m│[0m    
  [34m│[0m                                                        [34m│[0m  [36m│[0m                                                                                  [36m│[0m    
  [34m│[0m                                                        [34m│[0m  [36m│[0m                                                                                  [36m│[0m    
  [34m│[0m                                                        [34m│[0m  [36m│[0m                                                                                  [36m│[0m    
  [34m│[0m                                                        [34m│[0m  [36m│[0m                                                                                  [36m│[0m    
  [34m╰────────────────────────────────────────────────────────╯[0m  [36m╰──────────────────────────────────────────────────────────────────────────────────╯[0m    
                                                                                                                                                      
  [90m╭──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮[0m    
  [90m│[0m                                                                                                                                              [90m│[0m    
  [90m│[0m                                          [1;32m↑↓[0m  [37mnavigate[0m    [1;34m⏎[0m  [37mselect[0m    [1;33mh[0m  [37mhelp[0m    [1;35mesc[0m  [37mback[0m    [1;31mq[0m  [37mquit[0m                                        [90m│[0m    
  [90m│[0m                                                                                                                                              [90m│[0m    
  [90m╰──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯[0m    